funnel/__init__.py

```diff
@@ -60,8 +60,8 @@
 lastuser.init_usermanager(UserManager(db, models.User, models.Team))
 
 baseframe.init_app(app, requires=['funnel'], ext_requires=[
-    ('codemirror-markdown', 'pygments'), 'toastr', 'baseframe-mui', 'fontawesome>=4.0.0',
```

Comment:
pygments is required on proposal view pages. It renders syntax highlighting in code samples. Pygments and CodeMirror will not be used on the same page as they represent separate read and edit views.

Reply:
Moved Pygments back inside `ext_requires`